applications/rubintv/templates/vault-secrets.yaml

```diff
@@ -7,3 +7,13 @@ metadata:
 spec:
   path: "{{ .Values.global.vaultSecretsPath }}{{ .Values.global.tsVaultSecretsPath }}/rubintv"
   type: "Opaque"
+---
+apiVersion: ricoberger.de/v1alpha1
+kind: VaultSecret
+metadata:
+  name: pull-secret
+  labels:
+    {{- include "rubintv.labels" . | nindent 4 }}
+spec:
+  path: "{{- .Values.global.vaultSecretsPath }}/pull-secret"
+  type: kubernetes.io/dockerconfigjson
```

Comment:
This looks good, we are not going to use it for the frontend deployment, but we should use it for the workers deployment. We need to first update the worker images though, from: `lsstts/rubintv-broadcaster:c0030_usdf` to `ts-dockerhub.lsst.org/rapid-analysis:c0036`.

`lsstts/rubintv-broadcaster:c0030_usdf` is too old! from almost 1 year ago: https://hub.docker.com/r/lsstts/rubintv-broadcaster/tags.